# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1523035248 25200
#      Fri Apr 06 10:20:48 2018 -0700
# Branch stable
# Node ID f18f4700aded054afaca50167aba8a272a894e7c
# Parent  ab3904f253c5ab296f1d7a6800adb0a9c8a55c2e
configure: override default project URL with https URL

* configure.ac: Call AC_INIT with fifth argument to set project URL.

diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -15,17 +15,18 @@ dnl Copyright (C) 1993-2018 John W. Eato
 ### GNU General Public License for more details.
 ###
 ### You should have received a copy of the GNU General Public License
 ### along with Octave; see the file COPYING.  If not, see
 ### <https://www.gnu.org/licenses/>.
 
 ### Initialize Autoconf
 AC_PREREQ([2.65])
-AC_INIT([GNU Octave], [4.3.0+], [https://octave.org/bugs.html], [octave])
+AC_INIT([GNU Octave], [4.3.0+], [https://octave.org/bugs.html], [octave],
+        [https://www.gnu.org/software/octave/])
 
 ### Declare version numbers
 
 dnl Note that the version number is duplicated here and in AC_INIT because
 dnl AC_INIT requires it to be static, not computed from shell variables.
 
 OCTAVE_MAJOR_VERSION=4
 OCTAVE_MINOR_VERSION=3
